.github/workflows/pr-integration.yml

```diff
@@ -0,0 +1,76 @@
+# Action for running client integration tests against PRs. This test uses local
+# server changes but runs a published version of the client, so it will not pick
+# up local client changes.
+
+name: PR Integration Tests
+on:
+  push:
+    branches: [ dev ]
+    paths-ignore:
+      - 'README.md'
+      - '.github/**'
+      - 'service/local-dev/**'
+  pull_request:
+    branches: [ '**' ]
+    # This is a required check for all PRs, so don't add a paths-ignore here.
+    # We want this to always run so that the check passes.
+jobs:
+  pr-integration-job:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:13
+        env:
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+        # Give Postgres time to start by doing the health check later.
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}-${{ hashFiles('**/*.gradle') }}
+          restore-keys: v1-${{ runner.os }}-gradle-${{ hashfiles('**/gradle-wrapper.properties') }}
+
+      - name: Make sure Postgres is ready and init
+        env:
+          PGPASSWORD: postgres
+        run: |
+          pg_isready -h localhost -t 10
+          psql -h localhost -U postgres -f ./common/postgres-init.sql
+
+      - name: Render GitHub Secrets
+        id: github-secrets
+        shell: bash
+        env:
+          # Copied to GH from vault location: "secret/dsde/firecloud/dev/common/firecloud-account.json"
+          DEV_FIRECLOUD_ACCOUNT: ${{ secrets.DEV_FIRECLOUD_ACCOUNT }}
+        run:
+          echo "$DEV_FIRECLOUD_ACCOUNT" >"./integration/src/main/resources/rendered/user-delegated-sa.json"
```

Comment:
I confirmed that this doesn't log the secret when it runs: 
![image](https://user-images.githubusercontent.com/23193756/129225182-52412cdd-742b-4c1a-98ce-224b3cecba65.png)

Reply:
The value of this secret isn't actually used yet since the `ServiceStatus` test doesn't require auth or interact with other services, but we'll need it in the near future to do these things. Also test runner has an issue where it requires this be defined even if it isn't using it, so I went ahead and set it up.